quickstarts/microsoft.kubernetes/fleet/private-hubful-fleet-deploy.json

```
@@ -0,0 +1,94 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vnet_name": {
+            "type": "String"
+        },
+        "fleets_priv_fleet_name": {
+            "type": "String"
+        },
+        "fleets_sp_object_id": {
+            "type": "String"
+        }
+    },
+    "variables": {},
+    "resources": [
+        {
+            "type": "Microsoft.Network/virtualNetworks",
+            "apiVersion": "2024-05-01",
+            "name": "[parameters('vnet_name')]",
+            "location": "eastus",
+            "properties": {
+                "addressSpace": {
+                    "addressPrefixes": [
+                        "192.168.0.0/16"
+                    ]
+                },
+                "privateEndpointVNetPolicies": "Disabled",
+                "subnets": [
+                    {
+                        "name": "subnet",
+                        "id": "[resourceId('Microsoft.Network/virtualNetworks/subnets', parameters('vnet_name'), 'subnet')]",
+                        "properties": {
+                            "addressPrefix": "192.168.0.0/24",
+                            "delegations": [],
+                            "privateEndpointNetworkPolicies": "Disabled",
+                            "privateLinkServiceNetworkPolicies": "Enabled"
+                        },
+                        "type": "Microsoft.Network/virtualNetworks/subnets"
+                    }
+                ],
+                "virtualNetworkPeerings": [],
+                "enableDdosProtection": false
```

wonder if this can be trimmed a bit for readability without it breaking (not sure precisely how much we can get rid of)

```suggestion
                "subnets": [
                    {
                        "name": "subnet",
                        "properties": {
                            "addressPrefix": "192.168.0.0/24"
                        }
                    }
                ]
```

---

it is trimmed without breaking